### ISSUE TITLE
Loss does not change

### ISSUE THREAD

**Issue by nikhildhinagar** [SaoYan/LearnToPayAttention#9]
Hello, Thank you very much for posting this implementation of the ‘LearnToPayAttention’ paper. I was hoping you can help me with an issue I am having when running the code. When I run model 1 with (or without) attention with the default hyperparameter settings on the github page (LR = 0.1 etc) on CIFAR100, the training loss and train/test accuracy does not seem to change. Training loss is stuck at around 4.6 and test accuracy is stuck at 1%. I tried pytorch 0.4.1 and 1.0.0. Any help would be greatly appreciated. Thanks.

**Comment by Gonglixin** [comment]
I got the same problem, have you addressed it?
